detox/src/utils/customConsoleLogger.js

```diff
@@ -46,9 +46,9 @@ function overrideConsoleMethods(console, bunyanLogger) {
   if (!console.__detox_log__) {
     const log = bunyanLogger;
 
-    override(console, 'log', log.info.bind(log));
+    override(console, 'log', log.debug.bind(log));
```

Comment:
Not entirely sure if this is a legit change. If users want `console.log` they should get console.log.

Reply:
It is absolutely their problem in their code if this clutters things.

Reply:
OK, that makes sense. Will revert this.

Reply:
Ah cool, you reverted.